.github/workflows/federal-data-ingestion.yml

```
@@ -0,0 +1,165 @@
+  LIMIT: ${{ github.event.inputs.limit || '100' }}
+
+jobs:
+  fetch-federal-data:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
```

actionlint (1.7.8)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)